__init__.py

```diff
@@ -86,6 +86,7 @@ def handle_idle(self, _):
             LOG.error(e)
 
         self.gui['rtl_mode'] = self.rtlMode
+        self.gui['dateFormat'] = "DMY"
```

Comment:
should come from mycroft.conf

https://github.com/OpenVoiceOS/ovos-core/blob/dev/mycroft/configuration/mycroft.conf#L35

Reply:
Any reason this would be null or missing?

```python
self.gui['dateFormat'] = self.config_core.get("date_format") or "DMY"
```